app/code/Bulbulatory/Recommendations/Model/ResourceModel/Table/Collection.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Model\ResourceModel\Table;
+
+class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection
+{
+    protected $_idFieldName = 'id';
+    protected $_eventPrefix = 'bulbulatory_recommendations_table_collection';
+    protected $_eventObject = 'table_collection';
+
+    protected function _construct()
+    {
+        $this->_init('Bulbulatory\Recommendations\Model\Table', 'Bulbulatory\Recommendations\Model\ResourceModel\Table');

```

Comment:
When you need a class name, it's better to use ::class instead of using class name as a string.
```suggestion
        $this->_init(Bulbulatory\Recommendations\Model\Table::class, Bulbulatory\Recommendations\Model\ResourceModel\Table::class);
```
PHP will resolve it to a string anyway but working with this construct is much more comfortable. Because we use classes instead of strings, many IDEs will offer eg. different formating or when you change the class it will change all occurrences of it automatically. If we want to check what class is this, we'll be able to go to it directly from here instead of searching for it manually.

Reply:
@TM18

> When you need a class name, it's better to use ::class instead of using class name as a string.

Fixed! Thanks for the advice!